roles/network_plugin/cilium/templates/cilium-ds.yml.j2

```diff
@@ -26,145 +37,184 @@ spec:
 {% if rbac_enabled %}
       serviceAccountName: cilium
 {% endif %}
+        - name: clean-cilium-state
```

Comment:
Looks like this was intended as an initContainer but it wasn't specified in an initContainer section of the spec

Reply:
You're right.